README.md

```diff
@@ -33,6 +33,7 @@ Miscellaneous
 - We can handle "save to wallet" result only from activity (https://issuetracker.google.com/issues/239603300)
 - Autocomplete [issue](https://issuetracker.google.com/u/1/issues/242575301)
 - Custom text spans need additional API for proper RTL & BiDi support [issue](https://issuetracker.google.com/u/1/issues/237289433)
+- Use BaseLazyColumn until compose LazyColumn ime overlap issue - https://issuetracker.google.com/issues/192043120#comment78 not fixed 
```

Comment:
let's please specify that we need to use it if we have TextField which can be at the bottom, simply put:
Use `BaseLazyColumn` if it contains `TextFields`...